pkg/thing/interactors/errors.go

```diff
@@ -20,4 +22,30 @@ var (
 
 	// ErrSensorsNotProvided error occur when thing's sensors is not provided
 	ErrSensorsNotProvided = errors.New("thing's sensors not provided")
+
+	// ==================== REGISTER THING ERRORS: ====================
```

Comment:
I suggest we remove these comments. All errors in this package are imported with `entities`, the need to comment which interactors use each error (note that some more than one interactor may use a same error) means that we have space for improvement. From my POV, we may either:

1. Further standardize the **error names** and **error comments** (as it is done in the [docker reference file](https://github.com/docker/distribution/blob/master/reference/reference.go#L40)) so that they are generic enough to be reused but specific enough that we know the context which they are used.
>For example, **all** errors which regard to **schema** should be in the format `ErrSchema...`, all **argument** errors should end with `NotProvided` or `Missing`, and so on.

2. Add a subdirectory for each interactor (just like we did we thing/delivery/amqp) and each of those are subpackages with their own `errors.go` file and repeated errors (compiler won't complain since they are unique packages).

Option 1 seems the best for now, as number 2 I'm not sure if its optimal (and it requires changing the project's structure), but from my understanding it seems fine.

Reply:
Definitely +1 on removing the comments and standardizing the error names and comments. Option 2 is a valid one, of course, but at least for me, it's an over-optimization that leads to an ugly directory tree and the result would be completely the opposite of improving our code.

Reply:
Done! Agree with @joaoaneto and I adopted the first standard you mentioned @lcbm